frontend/public/components/graphs/_graphs.scss

```diff
@@ -56,44 +56,39 @@
   width: 100%;
 }
 
-.query-browser__kebab {
-  position: relative;
-}
-
 .query-browser__loading {
   width: 20px;
 }
 
 .query-browser__metrics-dropdown {
-  width: 200px;
-}
-.query-browser__metrics-dropdown-menu {
-  max-height: 25vh;
-  overflow: scroll;
+  .query-browser__metrics-dropdown-button {
```

Comment:
Avoid nesting when possible with BEM

Reply:
Nesting removed by https://github.com/openshift/console/pull/1881 and https://github.com/openshift/console/pull/1896